### PR TITLE
adb-install: add -d example

### PR DESCRIPTION
**Pull request by HUGHNew**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/adb-install.md [hunk 1]
```diff
@@ -15,7 +15,11 @@
 
 `adb install -r {{path/to/file.apk}}`
 
-- Grant all permissions listed in the app manifest:
+- Allow version code downgrade (debuggable packages only):
+
+`adb install -d {{path/to/file.apk}}`
+
+- Grant all permissions listed in the app manifest(is able to grant some system permissions like `android.permission.WRITE_SECURE_SETTINGS`):
```
Comment (CleanMachine1) [inline pages/common/adb-install.md:22]:
```suggestion
- Grant all permissions listed in the app manifest:
```

**Review by einverne — APPROVED** [review]
Thank you @HUGHNew for the new example, it looks good to me after @CleanMachine1 's suggestion.

**Review by navarroaxel — APPROVED** [review]

File: pages/common/adb-install.md [hunk 2]
```diff
@@ -15,6 +15,10 @@
 
 `adb install -r {{path/to/file.apk}}`
 
+- Allow version code downgrade (debuggable packages only):
```
Comment (CleanMachine1) [inline pages/common/adb-install.md:18]:
```suggestion
- Enable the function to use previous code (debuggable packages only):
```

I don't really know if this is correct, what do you think @HUGHNew

Comment (navarroaxel) [reply pages/common/adb-install.md:18]:
```suggestion
- Push an Android application allowing version code downgrade (debuggable packages only):
```

Comment (HUGHNew) [reply pages/common/adb-install.md]: I choose the better one in my opinion.

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**